pkg/zk/generators.go

```diff
@@ -142,10 +142,10 @@ func makeZkPodSpec(z *v1beta1.ZookeeperCluster, volumes []v1.Volume) v1.PodSpec
 				Exec: &v1.ExecAction{Command: []string{"zookeeperLive.sh"}},
 			},
 		},
-		VolumeMounts: []v1.VolumeMount{
+		VolumeMounts: append(z.Spec.VolumeMounts, []v1.VolumeMount{
```

Comment:
Is this Spec.Volume Mountsinitialized by default?

Reply:
Spec.VolumeMounts will be null by default and will not affect the original inherent volumeMounts of zk container. Only when we set values of spec.volumeMount in our charts, this will take effect and merge with original inherent volumemounts of zk container.

Reply:
> Is this Spec.VolumeMounts initialized by default?

Spec.VolumeMounts will be null by default and will not affect the original inherent volumeMounts of zk container. Only when we set values of spec.volumeMount in our charts, this will take effect and merge with original inherent volumemounts of zk container.